web/src/test/java/com/epam/rft/atsy/mapper/PasswordValidationMessageKeyMapperTest.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.rft.atsy.mapper;
```

Comment:
This test class stray a little bit. The PasswordValidationMessageKeyMapper class is in com.epam.rft.atsy.web.mapper package, but this test class is in com.epam.rft.atsy.mapper package.